package.json

```diff
@@ -13,27 +13,31 @@
   },
   "dependencies": {
     "aws-sdk": "^2.1.49",
-    "mime": "^1.3.4"
+    "mime": "^2.3.1"
   },
   "devDependencies": {
-    "browserify": "11.0.1",
+    "browserify": "16.2.2",
+    "babelify": "^9.0.0",
+    "@babel/core": "^7.0.0-beta.51",
+    "@babel/preset-env": "^7.0.0-beta.51",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Should we use the latest stable version of Babel instead of a beta version?

Reply:
Babelify implies/recommends installing the latest versions (which are the beta versions) in their readme:

https://github.com/babel/babelify

babel-core might not be needed though since we are using @ babel/core. Let me check this for both the skin and this repo.

Reply:
Oh ok, that's weird